ansible/roles/cgroups/vars/ubuntu-24.04.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Copyright (C) SchedMD LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cgroup_packages:
+- libcgroup2
+- libcgroup-dev
+- libpam-cgroup
+- libdbus-1-dev
+
+cgroup_grub_cmdline_linux:
```

Comment:
I would review these settings with SchedMD, but we do not have to block the PR on it.

Reply:
Will follow up to review these cgroups settings.